### PR TITLE
Move major logging to Kayvee logger

### PR DESCRIPTION
**Pull request by mohit**
This is just an attempt to move things over to `Kayvee.Logger`. Currently sphinx uses a custom kayvee format that has since been deprecated. 

I could move other things, but this seemed like a good start. 

cc: @dfuentes does this make sense from a common request logging format?

### CODE REVIEW

**Comment by dfuentes** [comment]
This seems to have parity with what was logged before.  I think it may also be useful to include HTTP method as well.

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
-		"github.com/Clever/sphinx/common",
-		"github.com/Clever/sphinx/config",
-		"github.com/Clever/sphinx/daemon",
-		"github.com/Clever/sphinx/handlers",
-		"github.com/Clever/sphinx/limit",
-		"github.com/Clever/sphinx/limitkeys",
-		"github.com/Clever/sphinx/matchers",
-		"github.com/Clever/sphinx/ratelimiter"
-	],
+	"GoVersion": "go1.5",
```
Comment (Christopher-Bradshaw) [inline Godeps/Godeps.json]: We are going back to 1.5?

File: handlers/http.go [hunk 2]
```diff
@@ -74,17 +64,22 @@ type httpRateLogger httpRateLimiter
...
+	rateLimitResponse := flattenRateLimitHeaders(getRateLimitHeaders(matches))
 	if err == leakybucket.ErrorFull {
-		log.Printf("[%s] BUCKET FULL", guid)
+		rateLimitResponse["limit"] = true
+	} else {
+		rateLimitResponse["limit"] = false
 	}
+	rateLimitResponse["guid"] = guid
+
+	common.Log.InfoD("request-finished", common.LogWithRequest(common.M{"guid": guid, "limit": true}, request))
```
Comment (Christopher-Bradshaw) [inline handlers/http.go]: Did you want to log `rateLimitResponse["limit"]` as the value for "limit" here?

File: common/logger.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	"strings"
+)
+
+var Log *logger.Logger
+
+func init() {
+	Log = logger.New("sphinx")
+}
+
+// M is an alias for map[Stringsing]interface{} to make log lines less painful to write.
```
Comment (Christopher-Bradshaw) [inline common/logger.go]: s/Stringsing/string/

File: common/logger.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+func init() {
+	Log = logger.New("sphinx")
+}
+
+// M is an alias for map[String]interface{} to make log lines less painful to write.
+type M map[string]interface{}
+
+// LogWithRequest concats the request to a given map[String]interface{} for use with Kayvee
+func LogWithRequest(data M, req Request) M {
```
Comment (Christopher-Bradshaw) [inline common/logger.go]: Function name slightly misleading as it doesn't actually do the logging. `ConcatWithRequest` maybe?

**Comment by Christopher-Bradshaw** [comment]
@mohit couple of minor comments.

File: handlers/http.go (hunk 2 above)
Comment (mohit) [inline handlers/http.go]: https://github.com/Clever/sphinx/pull/55/commits/985ea2a7da25856b5cb8566dd7214cc7417e1246

File: common/logger.go (hunk 4 above)
Comment (mohit) [inline common/logger.go]: https://github.com/Clever/sphinx/pull/55/commits/985ea2a7da25856b5cb8566dd7214cc7417e1246

**Pull request merged**